com.gitlab.cunidev.Gestures.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.gitlab.cunidev.Gestures",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gestures",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=~/.config",
```

Comment:
Is it for accessing libinput-gestures.conf? Would `xdg-config/libinput-gestures.conf:create` make more sense here?

Reply:
Yes. libinput-gestures.conf and libinput-gestures.conf.bak (arbitrary name for backup files) are the two ones needed in .config. Was leaving it broad in case further files were needed, but they could be restricted for the time being

Reply:
I'd rather have it limited to specific files rather than blank ~/.config access. It's a big hole I'd say.

Reply:
It seems to me like the permission you suggested creates a new directory named libinput-gestures.conf rather than a file in that position.

The program calls simply `f = open(self.filePath, "w+")` so the issue is in the Flatpak rather than the program itself